logprep/processor/requester/rule.py

```diff
@@ -0,0 +1,186 @@
+"""
+Requester
+=========
+
+The Requester is configured by the keyword :code:`requester`.
+It can be used to trigger external systems via web request or enrich eventdata by external
+apis.
+
+A speaking example for event enrichment via external api:
+
+..  code-block:: yaml
+    :linenos:
+    :caption: Given requester rule
+
+    filter: 'domain'
+    requester:
+      url: https://internal.cmdb.local/api/v1/locations
+      method: POST
+      content_type: application/json
+      target_field: cmdb.location
+      header:
+        Authorization: Bearer askdfjpiowejf283u9r
+      json:
+        hostname: ${message.hostname}
+    description: '...'
+
+..  code-block:: json
+    :linenos:
+    :caption: Incoming event
+
+    {"message": {"hostname": "BB37293hhj"}}
+
+..  code-block:: json
+    :linenos:
+    :caption: Processed event
+
+    {"message": {"hostname": "BB37293hhj"},
+     "cmdb": {
+         "locaction": {
+             "city": "Montreal",
+             "Building": "L76",
+             "Floor": 3,
+             "Room": 34
+             }
+        }
+    }
+"""
+import inspect
+import json
+import re
+
+import requests
+from attrs import define, field, validators
+
+from logprep.processor.calculator.rule import FIELD_PATTERN
```

Comment:
I'm not sure if I like this calculator dependency. Maybe we can move the `FIELD_PATTERN` up into the parent class? As I do use a similar pattern also in the `timestamp_differ`. Maybe we can align them and make it more general.

Reply:
moved FIELD_PATTERN to field_manager and refactored all three processors to this